.github/workflows/stale.yml

```
@@ -0,0 +1,31 @@
+name: "Stale PR Handler"
+
+on:
+  schedule:
+    - cron: "0 6 * * *"
```

I don’t think it’s the nicest of thing to do this over the weekend.

```suggestion
    - cron: "0 6 MON-THU * *"
```

---

Good point. it would be a bit mean to flag a PR as stale on Friday then close it on Monday!